CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+cmake_minimum_required(VERSION 3.1)
+project(lodepng)
+
+# define target to install and link tests against
+add_library(${CMAKE_PROJECT_NAME}
```

Comment:
Also you can use `option(BUILD_SHARED_LIBS "Build ${CMAKE_PROJECT_NAME} as shared library"  OFF)` if you care about type of libbrary to being build